packages/vite-plugin/jest.config.ts

```diff
@@ -23,7 +26,8 @@ const config: Config = {
             optimizer: {
               globals: {
                 vars: {
-                  __PACKAGE_VERSION__: packageJson.version,
+                  __PACKAGE_VERSION__: JSON.stringify(packageJson.version),
```

Comment:
l: Could we extract this into a common jest config that is shared among the plugins?

Reply:
Again, will do this in my follow up PR that migrates from Jest to Vitest.